build.gradle

```diff
@@ -351,3 +351,20 @@ subprojects {
   }
   /* INCLUDED PROJECT DEPENDENCY HELPER */
 }
+
+tasks.register("dev") {
+  subprojects.each { subproject ->
+    subproject.tasks.check.dependsOn subproject.tasks.googleJavaFormat
+    dependsOn subproject.tasks.check
+  }
+}
+
+tasks.register("devFull") {
+  if (!System.getenv('JIB_INTEGRATION_TESTING_PROJECT')) {
```

Comment:
`... && !System.getenv('JIB_INTEGRATION_TESTING_LOCATION')) {`

This is documented in `CONTRIBUTING.md`.

Reply:
That's optional though isn't it?

Reply:
Defining only one of them is enough to run `./gradlew integrationTest`. https://github.com/GoogleContainerTools/jib/blob/9d84f949a5419fb89cb809cee67ebdb6223e8648/jib-core/src/integration-test/java/com/google/cloud/tools/jib/IntegrationTestingConfiguration.java#L33-L35

Reply:
Oh yeah you're right. I'll fix this

Reply:
Used same message.